src/pages/admin/team/components/members-list/MembersList.scss

```
@@ -16,15 +19,15 @@
     }
 
     &-categories {
-        border-bottom: 1px solid #a4a49b;
-        margin-bottom: 10px;
-        margin-left: 10px;
-        margin-right: 10px;
+        border-bottom: 1px solid colors.$grey-700;
+        margin-bottom: 0.625rem;
+        margin-left: 0.625rem;
+        margin-right: 0.625rem;
```

This can be simplified as follows:

```suggestion
        margin: 0 0.625rem 0.625rem;
```

---

Your suggestion was implemented! Thank you for your time